### DISCUSSION TITLE
[Art competition] Perfect Indistinguishability

### DISCUSSION THREAD

**Discussion by presenthee** [prosyslab-classroom/cs348-information-security#413] [Art competition]
![DALL·E 2023-05-31 00 55 59](https://github.com/prosyslab-classroom/cs348-information-security/assets/77828537/a58b9446-bc15-4e66-86f7-c9c91416a8bf)

## Short Description
For me, [Perfect Indistinguishability] means [People looking at a giant chair floating in the sky, René Magritte style painting].

## Long Description (up to 3 sentences)
When I saw Perfect Indistinguishability, I think of Plato's Idea. Practical schemes we see are the result of an effort to copy the ideal encryption scheme, the Idea. I represent this as people looking at a chair in the sky, using Surrealism style.

**Comment by KihongHeo** [comment]
Interesting. But what "chair in the sky" does mean?
